### ISSUE TITLE
Can't put book in bookcase

### ISSUE THREAD

**Issue by Rhemora** [talonos2/BlightfallTracker#207]
I have created a blank book (In order to start up the thaumcraft quest string) however it seems that blank books cannot be put in the book case, I can put my quest book, cooking book etc in the book case but it will not allow me to do so with the blank book.

**Comment by Raclex** [comment]
I don't think you can put any stackable items in bookcases. Probably how the mod works.

**Comment by talonos2** [comment]
Nah, TiC books stack and go in bookshelves. Bibliocraft just balcklisted
blank books for some reason.

**Comment by Rhemora** [comment]
Exactly which means that you cannot do the quest string which kinda blocks off a whole side of the game.

**Comment by talonos2** [comment]
How so?

If you're trying to make a thaumonomicon, right click a _vanilla_ bookshelf
with a wand. Bibliocraft bookshelves have not ever worked.

Otherwise, I don't see what quest is blocked by failure to put blank books
in bookshelves.

**Comment by Raclex** [comment]
Agreed

**Comment by Rhemora** [comment]
Because all this has been WITH a vanilla bookshelf not with a bibliocraft bookshelf.

**Comment by talonos2** [comment]
You cannot put _any_ books in a vanilla bookshelf, blank or otherwise. They
are decorative blocks that interact with other specific blocks such as
enchantment tables. http://minecraft.gamepedia.com/Bookshelf The
"Bookcases" that you can put books in are bibliocraft, and do not interact
with wands in a way that generates Thaumonomicons.

If you want to put blank books in book-cases, I think you can do so by
changing an option in bibliocraft.cfg in the modpack folder that is off by
default. I cannot think of any quests that depend on being able to put
blank books in bookshelves. If there are, please mention it to me.
